code/datums/autolathe/autolathe.dm

```diff
@@ -2,7 +2,14 @@ var/datum/category_collection/autolathe/autolathe_recipes
 
 /datum/category_item/autolathe/New()
 	..()
-	var/obj/item/I = new path()
+	var/obj/item/I
+	if(path)
+		I = new path()
+
+	if(!I)	// Something has gone horribly wrong, or right.
+		log_debug("[name] created an Autolathe design without an assigned path. This is expected for only the Material Sheet generation.")
```

Comment:
If it's expected for something then that something should suppress the log

Reply:
These only fire off for Materials once per type, and wherein they actually have sheets. I think it's safer to have it remain for them incase something explodes later on for autolathes, in the manner of a canary.